public/app/plugins/panel/table/column_options.ts

```diff
@@ -78,6 +87,7 @@ export class ColumnOptionsCtrl {
       dateFormat: 'YYYY-MM-DD HH:mm:ss',
       thresholds: [],
       mappingType: 1,
+      align: '',
```

Comment:
Maybe this should be more explicit like 'default' instead?

Reply:
'default' it is.